orchagent/fdborch.cpp

```diff
@@ -1121,15 +1135,15 @@ void FdbOrch::updateVlanMember(const VlanMemberUpdate& update)
 }
 
 bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name,
-        FdbData fdbData)
+        FdbData fdbData, bool check_vlan_member)
```

Comment:
Do we need to change the API signature ? If there is a need to check for vlan member it can be done using the isDipTunnelSupported call ?

Reply:
Removed changes to API signature.